qemu/tests/cfg/ovs_host_vlan.cfg

```diff
@@ -0,0 +1,43 @@
+- ovs_host_vlan:
+    type = ovs_host_vlan
+    private_vlan_bridge = "ovs_private_br"
+    only Linux
+    bridge_ip = 192.168.10.1/24
+    ovs_br_create_cmd = "ovs-vsctl br-exists ${private_vlan_bridge} || ovs-vsctl add-br ${private_vlan_bridge};"
+    ovs_br_create_cmd += "ifconfig ${private_vlan_bridge} ${bridge_ip} up"
+    ovs_br_remove_cmd = "ifconfig ${private_vlan_bridge} down; ovs-vsctl del-br ${private_vlan_bridge}"
+    qemu_sandbox = off
+    vms = "vm1 vm2 vm3 vm4"
+    image_snapshot = yes
+    start_vm = no
+    ip_nic1_vm1 = 192.168.10.11
+    ip_nic1_vm2 = 192.168.10.12
+    ip_nic1_vm3 = 192.168.10.21
+    ip_nic1_vm4 = 192.168.10.22
+    nic_ifname_vm1 = ovs_vm1
+    nic_ifname_vm2 = ovs_vm2
+    nic_ifname_vm3 = ovs_vm3
+    nic_ifname_vm4 = ovs_vm4
+    ovs_port_vlan_vm1 = 10
+    ovs_port_vlan_vm2 = 10
+    ovs_port_vlan_vm3 = 20
+    ovs_port_vlan_vm4 = 20
+    kill_vm = no
+    kill_vm_gracefully = no
+    kill_unresponsive_vms = no
+    setup_cmd = cd %s && rm -rf netperf-2.6.0 && tar xvfj netperf-2.6.0.tar.bz2 && cd netperf-2.6.0 && ./configure --enable-burst --enable-demo=yes && make
+    netperf_link = netperf-2.6.0.tar.bz2
+    netserver_cmd = netperf-2.6.0/src/netserver
+    netperf_cmd = netperf-2.6.0/src/netperf -l %s -H %s
+    netperf_path = "/var/tmp/"
+    file_transfer_test = yes
+    Windows:
```

Comment:
Please drop windows part if not really supported.

Reply:
Windows will be supported later on. So I'll keep the handler here.

Reply:
@xiywang It is okay, would you add a 'TODO' comment at all windows implementation part then?

Reply:
Fair enough. ;)